sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/DescribeTableSuite.scala

```diff
@@ -83,4 +88,524 @@ class DescribeTableSuite extends v1.DescribeTableSuiteBase with CommandSuiteBase
           Row("Partition Provider", "Catalog", "")))
     }
   }
+
+  test("DESCRIBE AS JSON throws when not EXTENDED") {
```

Comment:
Can we move these tests to `org.apache.spark.sql.execution.command.v1.DescribeTableSuite`? Then they will be run twice with and without Hive.